flow/util/utils.mk

```diff
@@ -145,6 +145,18 @@ ifdef GDB
 OPENROAD_EXE := gdb --args $(OPENROAD_EXE)
 endif
 
+## Convert RVE DRC database to JSON
+convert_rve: $(OBJECTS_DIR)/drc.json
+
+$(OBJECTS_DIR)/drc.json: $(DRC_FILE)
```

Comment:
Where is DRC_FILE defined or is the user supposed to supply it?

Reply:
it would need be user defined, thats why I have the message below about needing to set it.
If there would be a standard place for it, I suppose it could be predefined.

Reply:
Is `DRC_FILE` different from `KLAYOUT_DRC_FILE`?

Reply:
Yes, it it's the file DRC results from another tool